src/Symfony/Component/Form/ChoiceList/Factory/PropertyAccessDecorator.php

```diff
@@ -218,12 +219,13 @@ public function createView(ChoiceListInterface $list, $preferredChoices = null,
             @trigger_error('Passing callable strings is deprecated since version 3.1 and PropertyAccessDecorator will treat them as property paths in 4.0. You should use a "\Closure" instead.', E_USER_DEPRECATED);
         }
 
+        // Deprecated since 3.3 and to be removed in 4.0 with the condition above
```

Comment:
if it is deprecated, it must trigger a deprecation notice.

And why deprecating this feature ? It is not consistent with other properties.